lib/primer/view_components/linters/argument_mappers/label.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative "base"
+
+module ERBLint
+  module Linters
+    module ArgumentMappers
+      # Maps classes in a label element to arguments for the Label component.
+      class Label < Base
+        SCHEME_MAPPINGS = {
```

Comment:
I'd love to be able to reuse these from the components, as copying them here will likely lead to them being out of date. What do you think?

Reply:
I'd love to have access to the component's constants here, but that'd require including the component in this file, which will basically load the whole Rails app.
I thought about having a separate `Constants` namespace that could hold that information and could be shared between the component definition and the linters, but that would make development harder

Reply:
@manuelpuyol gotcha. Can you describe the overhead and downsides of loading the whole Rails app?

Reply:
Since we rely on autoloading to load our components (and helpers), we can't simply require the components and be good to go. We'd need to manually load everything needed - ViewComponent framework, component, helpers...
That can get very complicated once the library grows.